docs(gallery): expands design guidelines for gallery layout. (#3671)
patternfly/patternfly-org: packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/gallery.md

id: Gallery
section: layouts
---
The **gallery** layout is used to arrange content in a responsive grid. Content will wrap responsively to create uniform rows and columns.

## Elements

> The gallery layout is designed so that all of its elements are of uniform size, display horizontally, and wrap responsively.

Author: ```suggestion
The gallery layout is designed so that all of its elements wrap responsively,are of uniform size, and are displayed horizontally.
```

![image](./img/gallery-example.png)

1. **Gallery:** The gallery container, which contains all gallery items.

> 1. **Gallery item:** And individual item to be placed in the gallery.

Author: ```suggestion
1. **Gallery item:** An individual item to be placed in the gallery.
```

## Usage

> While a gallery layout resembles a grid, it is not the same as the grid layout.

Author: ```suggestion
While a gallery layout resembles a grid, it is not the same as the grid layout, which is structured more rigidly.
```

> Common use cases for the gallery layout include card views.

Author: ```suggestion
The gallery layout is commonly used for UIs that contain multiple card components.
```

## Variations

### Gutters around items

To adjust spacing between items, you can apply gutters.

![image](./img/gallery-example-gutter.png)

> ### Items that customized wrapping

Author: ```suggestion
### Items with customized wrapping
```

> Both a minimum and maximum width can be defined for gallery items. These values determine the width that a gallery item is allowed to be before they wrap.

Author: ```suggestion
Both a minimum and maximum width can be defined for gallery items. These values determine the width that a gallery item is allowed to be before wrapping.
```
